packages/host/app/components/operator-mode/code-submode/playground/playground-content.gts

```diff
@@ -199,6 +194,32 @@ export default class PlaygroundContent extends Component<Signature> {
   @tracked private fieldChooserIsOpen = false;
   @tracked private cardResource: ReturnType<getCard> | undefined;
 
+  private get specQuery(): Query {
+    return {
+      filter: {
+        on: specRef,
+        eq: { ref: this.args.codeRef },
+      },
+    };
+  }
+
+  private searchFieldSpec = getSearch(
```

Comment:
since this is in the constructor, you'll need to use the new `{{consumeContext}}` helper to get access to `getCards()`